app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -162,14 +162,17 @@ class BrowserFragment : Fragment(), BackHandler, CoroutineScope {
             }
         }
 
-        view.engineView.asView().apply {
-            (layoutParams as CoordinatorLayout.LayoutParams).apply {
-                setMargins(
-                    0,
-                    0,
-                    0,
-                    (resources.displayMetrics.density * TOOLBAR_HEIGHT).toInt() +
-                        QUICK_ACTION_SHEET_HANDLE_HEIGHT)
+        if (customTabSessionId == null) {
```

Comment:
Maybe outside of the scope of this issue, but we can move this to the CustomTabsIntegration class so we don't have to do this check multiple times in different parts of BrowserFragment.

Feel free to ignore this and I'll file a follow up issue for it.